sparkplug_plugin/sparkplug_b_input.go

```
@@ -0,0 +1,1135 @@
+	case SparkplugDataTypeString:
+		return "String"
+	case SparkplugDataTypeDateTime:
+		return "DateTime"
+	case SparkplugDataTypeText:
+		return "Text"
+	default:
+		return "Unknown"
+	}
+}
+
+func (s *sparkplugInput) payloadToJSON(payload *sproto.Payload) []byte {
```

golangci-lint (1.64.8)

func `(*sparkplugInput).payloadToJSON` is unused

(unused)